### PR TITLE
fix(w3c/headers): prefix spec status correctly

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: src/w3c/headers.js [hunk 1]
```diff
@@ -413,7 +413,6 @@ export async function run(conf) {
     conf.logos.push({ ...baseLogo, ...teamSubmissionLogo });
   }
   conf.isSubmission = conf.isMemberSubmission || conf.isTeamSubmission;
-  conf.anOrA = precededByAn.includes(conf.specStatus) ? "an" : "a";
```
Comment (sidvishnoi) [inline src/w3c/headers.js:416]: Nit: `precededByAn` is unused now.

Comment (sidvishnoi) [reply src/w3c/headers.js:416]: Oh wait. We've a linter. 😄

Comment (marcoscaceres) [reply src/w3c/headers.js:416]: lol

File: tests/spec/w3c/headers-spec.js [hunk 2]
```diff
@@ -136,6 +136,34 @@ describe("W3C — Headers", () => {
       expect(firstDt.textContent).toContain("Latest editor's draft:");
     });
 
+    it("prefixes status text properly", async () => {
+      const edDoc = await makeRSDoc(
+        makeStandardOps({
+          shortName: "test",
+          specStatus: "ED",
```
Comment (sidvishnoi) [inline tests/spec/w3c/headers-spec.js:143]: Test in a for loop with some more specStatus?

Comment (marcoscaceres) [reply tests/spec/w3c/headers-spec.js:143]: was going to, but the text is different.

**Review by sidvishnoi — APPROVED** [review]
Nice

File: tests/spec/w3c/headers-spec.js (hunk 2 above)
Comment (sidvishnoi) [reply tests/spec/w3c/headers-spec.js:143]: Can use `match` on text.

Comment (sidvishnoi) [reply tests/spec/w3c/headers-spec.js:143]: Or never mind. YOLO.

**Pull request merged**